subsys/mpsl/init/mpsl_init.c

```diff
@@ -267,12 +358,24 @@ static int32_t mpsl_lib_init_internal(void)
 	clock_cfg.rc_ctiv = 0;
 	clock_cfg.rc_temp_ctiv = 0;
 #endif
+#else
```

Comment:
Isn't it fine to leave the clock_cfg as-is? Atm it just gets ignored in MPSL on 54h

Otherwise: we can use less ifdef logic by initializing clock_cfg as:

`mpsl_clock_lfclk_cfg_t clock_cfg = {};`

Reply:
Maybe that is simpler. The clock handling is anyway being refactored now so this will have to change in the future anyway.

Reply:
The `CLOCK_CONTROL_*` configs are not defined since `CONFIG_CLOCK_CONTROL` is not set so there will be some ifdeffery anyway. Propose to leave it as is for now.